aurget

```diff
@@ -333,6 +344,7 @@ set_defaults() {
   sync_mode='install'
   temp_directory='/tmp/aurget'
   user_config="${XDG_CONFIG_HOME:-$HOME/.config}/aurgetrc"
+  pkg_cache_dir=$((grep -m 1 '^CacheDir' /etc/pacman.conf || echo 'CacheDir = /var/cache/pacman/pkg') | sed 's/CacheDir = //')
```

Comment:
I don't have a PACMAN_CONF env variable and I couldn't find any mention on pacman.conf man page either.

Reply:
Right, this would be something we'd invent for our own use case, to make this configurable. It's not a super compelling use-case but it makes it much easier to test.